### DISCUSSION TITLE
what significance is 16000 sample blocksize and lpc order mainly for cd ripping (for best quality flac)

### DISCUSSION THREAD

**Discussion by impotentfield** [enzo1982/freac#293] [Audio Format Discussion]
im not sure of the significance of 16000 sample blocksize and lpc and quantization slider values. when i do a cd rip copy and use customization, is this a good setting where the quality stands as it being proper for quality ? ... or do i have to experiment and there is not really an experienced answer for this sort of thing. there are no sources from which i can get information on the internet so far. i would appreciate if i could have some clarity on this subject.

**Comment by enzo1982** [comment]
FLAC is a lossless format, so the settings do not affect quality at all. All they do is to influence compression efficiency.

Generally, larger block sizes as well as higher LPC and quantization precision values can yield slightly better compression, but increase the time it takes to compress a file.
